Doc/library/socket.rst

```diff
@@ -928,6 +928,9 @@ to sockets.
       :exc:`OSError` is now raised if an error occurs when the underlying
       :c:func:`close` call is made.
 
+   .. versionchanged:: 3.7
```

Comment:
I think it's misleading to add a "versionchanged" here. It's just a regression that we are fixing.